npm/qsharp/src/main.ts

```diff
@@ -91,4 +91,9 @@ export function getLanguageServiceWorker(): ILanguageServiceWorker {
   );
 }
 
+export function collectTestCallables(config: IProgramConfig): string[] {
+  ensureWasm();
+  return wasm!.collect_test_callables(config);
+}
+
```

Comment:
You shouldn't need these changes in this file  - `Compiler`/`ICompiler` are already reexported through here (which, for context, is the Node.js specific module entrypoint, and is only used in tests)